src/springfield/entity.py

```diff
@@ -267,7 +273,7 @@ def __len__(self):
         return len(self.__values__)
 
     def iteritems(self):
-        return self.__values__.iteritems()
+        return self.__values__.items()
```

Comment:
nitpick: `iteritems` should probably be marked as deprecated.

For py2 this does not actually return an iterator. That could break something...